layouts/partials/footer.html

```diff
@@ -14,6 +14,7 @@
             <li class="sitemap-head">IPFS</li>
             <li><a href="https://ipfs.io/#install">Install</a></li>
             <li><a href="https://github.com/ipfs/ipfs">GitHub</a></li>
+            <li><a href="https://github.com/ipfs/community/blob/master/code-of-conduct.md”>Code of Conduct</a></li>
```

Comment:
```suggestion
            <li><a href="https://github.com/ipfs/community/blob/master/code-of-conduct.md">Code of Conduct</a></li>
```
Evil curly quote breaks stuff 😊